### PR TITLE
Update README SDK crate list

### PR DESCRIPTION
**Pull request by johnbatty**
Updated the README SDK crate list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,14 @@ These crates are available from in (sdk):
 - azure_core
 - azure_identity
 - azure_cosmos
```
Comment (cataggar) [inline README.md:13]: We do need to update this to azure_data_cosmos. #482. I'll have a look today.

File: README.md [hunk 2]
```diff
@@ -11,8 +11,14 @@ These crates are available from in (sdk):
...
-- azure_storage
+- azure_data_tables
+- azure_iot_hub
+- azure_messaging_eventgrid
```
Comment (cataggar) [inline README.md:16]: I'd prefer to keep eventgrid out the readme until #254.

File: README.md [hunk 3]
```diff
@@ -11,8 +11,14 @@ These crates are available from in (sdk):
...
+- azure_messaging_servicebus
```
Comment (cataggar) [inline README.md:17]: Same with servicebus, until #255.

**Review by cataggar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (johnbatty) [reply README.md]: I see this rename has just happened, so have updated to azure_data_cosmos

File: README.md (hunk 2 above)
Comment (johnbatty) [reply README.md]: Removed

File: README.md (hunk 3 above)
Comment (johnbatty) [reply README.md]: Removed

**Review by cataggar — APPROVED** [review]

**Pull request merged**